libcontainer/factory_linux.go

```diff
@@ -380,6 +381,29 @@ func (l *LinuxFactory) StartInitialization() (err error) {
 		return fmt.Errorf("unable to convert _LIBCONTAINER_LOGPIPE: %w", err)
 	}
 
+	// Get mount files (O_PATH).
+	// We assume in other parts that mountFiles is nil when there are no fds to mount.
+	var mountFiles []*os.File
+	fdsStr := os.Getenv("_LIBCONTAINER_MOUNT_FILE_FDS")
+	if fdsStr != "" {
+		fds := strings.Split(fdsStr, ";")
+		mountFiles = make([]*os.File, len(fds))
+		for i, fd := range fds {
+			if fd == "" {
+				continue
+			}
+
+			mountFileFd, err := strconv.Atoi(fd)
+			if err != nil {
+				return fmt.Errorf("unable to parse _LIBCONTAINER_MOUNT_FILE_FDS(%q), %q: %w", fdsStr, fd, err)
```

Comment:
no need to print value of `fd` here, as an error message from `strconv.Atoi` already have it.

Reply:
Right, removing. Thanks!